src/components/ui/AppQrCode.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <qrcode-vue
```

Comment:
A (white) border around the QR code would be nice, my phone failed to pick it up because the position marks had too low of a contrast compared to the background. The qrcode.vue docs mention a [`margin` property](https://github.com/scopewu/qrcode.vue#margin), but it seemed to not work in my testing. Giving it an 8px CSS padding with a background-color of white seemed to work fine.